.github/workflows/cicd.yml

```
@@ -12,14 +12,16 @@ on:
 jobs:
   test:
     name: Python CI
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
 
     steps:
     - uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)